### PR TITLE
Schedule revamp

### PR DESCRIPTION
**Pull request by plaindocs**
All I wanted to do was add the job fair...  :ogre:

### CODE REVIEW

File: docs/conf/portland/2018/schedule.rst [hunk 1]
```diff
@@ -7,88 +7,152 @@ Schedule
...
 ---------------
 
+Hike
+~~~~
+
 The only event scheduled on Saturday is the :doc:`annual hike to Pittock Mansion </conf/portland/2018/hike>`.
 If you get into town early, join us on the hike and take the chance to explore Portland in all of its glory.
 
-The hike starts at **2pm** on Saturday.
+where
```
Comment (ericholscher) [inline docs/conf/portland/2018/schedule.rst]: Not in love with the lowercase styles here. Is there a reason not to just use either lists, or rst sections?

**Review by ericholscher — COMMENTED** [review]
❤️ -- looks good with some formatting tweaks.

File: docs/conf/portland/2018/schedule.rst (hunk 1 above)
Comment (ericholscher) [reply docs/conf/portland/2018/schedule.rst]: I kinda did that here: http://www.writethedocs.org/conf/portland/2018/sponsors/prospectus/#job-fair -- with the "Logisitics" and "Layout" bits.

Comment (plaindocs) [reply docs/conf/portland/2018/schedule.rst]: These are definition lists which need CSS to look ok (also need to degreen the titles)

Aye on lower case

Comment (ericholscher) [reply docs/conf/portland/2018/schedule.rst]: Probably something we should standardize on and put in the style guide.

**Pull request merged**